superset/jinja_context.py

```diff
@@ -252,6 +270,8 @@ def latest_sub_partition(self, table_name, **kwargs):
             table_name=table_name, schema=schema, database=self.database, **kwargs
         )
 
+    latest_partition = first_latest_partition
```

Comment:
@villebro `latest_partition` is still supported  (though not preferred) per ^^.

Reply:
@john-bodley I merely meant that it would be good to update the example in `docs/sqllab.rst` to reference the new preferred method: https://github.com/apache/incubator-superset/blame/master/docs/sqllab.rst#L59